walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,126 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.HashSet;
+import java.util.Set;
+
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    private static int start;
+    private static int end;
+    private static TimeRange event_time;
+    private static TimeRange event_1;
+    private static TimeRange event_2;
+
+  Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+	long current_duration = request.getDuration();
+    Collection<String> request_attendees = request.getAttendees();
+    String only_event_attendee;
+    Collection<TimeRange> Event_Times = new ArrayList<>();
```

Comment:
Please follow Java naming conventions.  I think variable names should be all lower cases.